sway-ir/src/context.rs

```diff
@@ -55,7 +55,12 @@ impl Context {
     ///
     /// The name will be in the form `"anon_N"`, where `N` is an incrementing decimal.
     pub fn get_unique_name(&mut self) -> String {
-        let sym = format!("anon_{}", self.next_unique_sym_tag);
+        format!("anon_{}", self.get_unique_id())
+    }
+
+    /// Get a globally unique symbol id.
+    pub fn get_unique_id(&mut self) -> u64 {
```

Comment:
Is this now used somewhere else?  If not, there's no real need to split a function up because we might want it one day, and to make it `pub`.  But I'm looking at only the most recent changes, so maybe you're using `get_unique_id()` elsewhere...